src/strategies/flexa-capacity-staking/index.ts

```diff
@@ -0,0 +1,31 @@
+import fetch from 'cross-fetch';
+import { getAddress } from '@ethersproject/address';
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'amptoken';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  return Object.fromEntries(await Promise.all(addresses.map(async (address) => {
```

Comment:
looks like this will send on API call for one addresses, it won't scale when there are more votes on the proposal, can u plz find an alternative to this? maybe to get all data in a single API call

Reply:
Hi @ChaituVR, thanks for the note. 

I was able to use a batched version of the API, and you can see the change here https://github.com/amptoken/snapshot-strategies/commit/920e5eb7419abe8aa01875c1f2eede67e5e4a41d

Please let me know if you need anything else.